### PR TITLE
Billing Api

### PR DESCRIPTION
**Pull request by ifedapoolarewaju**
Implements transloadit billing api.

Very little changes made with this one. 😃

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 * Fix compatibility to Ruby >=2.1 and Rails 5.
 * Remove bored instance logic (thanks @ifedapoolarewaju for the PR). This shouldn't affect users at all and removes
   the need for another HTTP request before the actual HTTP request.
+* We now have `transloadit.bill` method to retrieve billing reports
```
Comment (tim-kos) [inline CHANGELOG.md]: There is a missing article *the* here. Sounds like bad grammar.

File: README.md [hunk 2]
```diff
@@ -212,6 +212,16 @@ transloadit.assembly(
...
 
+### 7. Getting Bill reports
+
+If you want to retrieve your transloadit account billing report for a particular month and year
+you can use the `bill` method passing the required month and year like the following:
+
+```ruby
+transloadit.bill(month=2, year=2016)
+```
+not specifying the `month` or `year` would default to the current month or year.
```
Comment (tim-kos) [inline README.md]: It should be *Not specifying* and not *not specifying*.

Also it should mention if January is number *1* or number *0*.

**Comment by tim-kos** [comment]
Reviewed this. Minor spelling changes required. Then I'll merge this. 👍

**Comment by ifedapoolarewaju** [comment]
@tim-kos corrected! 👍

File: lib/transloadit.rb [hunk 3]
```diff
@@ -76,6 +77,25 @@ def assembly(options = {})
...
   #
+  # Gets user billing reports for specified month and year.
+  # Defaults to current month or year if corresponding param is not specified.
+  #
+  # @param [Integer] month the month for which billing reports should be retrieved.
+  #   defaults to current month if not specified.
+  # @param [Integer] year the year for which billing reports should be retrieved.
+  #   defaults to current year if not specified.
+  #
+  def bill(month = nil, year = nil)
```
Comment (rmehner) [inline lib/transloadit.rb]:
```ruby
def bill(month = Date.today.month, year = Date.today.year)
```

would save you from the nil checks and would feel more ruby like :)

File: lib/transloadit.rb [hunk 4]
```diff
@@ -76,6 +77,25 @@ def assembly(options = {})
...
+    today = Date.today
+    # convert month to 2 digit  format
+    month = format('%02d', month || today.month)
+    year  = today.year if year.nil?
```
Comment (rmehner) [inline lib/transloadit.rb]: :art: indentation is a little off here, year only needs once space before the equal sign. Yep, we should automate that style check somewhen :)

**Pull request merged**